### PR TITLE
Bumped Kano OS version to 2.4.0

### PR DESCRIPTION
**Pull request by radujipa**

### CODE REVIEW

**Comment by radujipa** [comment]
cc @alex5imon @tombettany @Ealdwulf @skarbat

File: kano_updater/version.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
 
-VERSION = "Kanux-Beta-2.3.0"
```
Comment (tombettany) [inline kano_updater/version.py]: Hmm, I'm not sure about this ;-)

**Comment by tombettany** [comment]
:shipit: :anchor:

File: kano_updater/version.py (hunk 1 above)
Comment (radujipa) [inline kano_updater/version.py]: STOP THE PRESSES!!!!

**Pull request merged**